type.proto

```diff
@@ -606,3 +607,7 @@ message PtObject {
     optional Trip trip = 20;
     repeated int32 scores = 21;
 }
+
+message FareZone {
+    required string name = 1;   
```

Comment:
I'd let it optional (id would come after and be required)... Any opinion @kinnou02 ?

Reply:
https://developers.google.com/protocol-buffers/docs/proto#required_warning